src/explorers/SliceSampler.jl

```diff
@@ -12,9 +12,10 @@ adapt_explorer(explorer::SliceSampler, _, _) = explorer
 explorer_recorder_builders(::SliceSampler) = [] 
 
 function step!(explorer::SliceSampler, replica, shared)
+    log_potential = find_log_potential(replica, shared)
```

Comment:
Agreed, I see no good reason to have this inside the loop

Reply:
@nikola-sur done

Reply:
Otherwise, everything looks good to me!